src/config/config.service.spec.ts

```diff
@@ -209,8 +204,9 @@ describe('ReconnectionConfigService', () => {
       expect(reconnectionConfigService.providerUserGraphEndpoint(ALL_ENV.PROVIDER_ID as string).toString()).toStrictEqual(ALL_ENV.PROVIDER_USER_GRAPH_ENDPOINT);
     });
 
-    it('should get provider api token', () => {
-      expect(reconnectionConfigService.providerApiToken(ALL_ENV.PROVIDER_ID as string).toString()).toStrictEqual(ALL_ENV.PROVIDER_ACCESS_TOKEN);
```

Comment:
I think we still want to keep the test that checks the token (if present) is properly returned

Reply:
I thought this was misnamed, but I see what it was meant for now. I'll re-add